src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -30,6 +32,14 @@ public ConsumesAttribute(string contentType, params string[] otherContentTypes)
                 throw new ArgumentNullException(nameof(contentType));
             }
 
+            // For validation purposes, preserves existing behavior.
```

Comment:
Changed with             // We want to ensure that the given provided content types are valid values, so
            // we validate them using the semantics of MediaTypeHeaderValue.